code/modules/paperwork/photography.dm

```diff
@@ -650,3 +651,33 @@
 			I.forceMove(F)
 		F.update_icon()
 	qdel(src)
+
+/obj/structure/sign/picture_frame/proc/display(obj/item/I, mob/user)
+	if(!user.transferItemToLoc(I, src))
+		return
+	framed = I
+	update_icon()
+
+
+/obj/structure/sign/picture_frame/persist
+	name = "durable picture frame"
+
+/obj/structure/sign/picture_frame/persist/Initialize()
+	. = ..()
+	GLOB.persist_frames += src
+
+/obj/structure/sign/picture_frame/persist/Destroy()
+	GLOB.persist_frames -= src
+	return ..()
+
+/obj/structure/sign/picture_frame/persist/display(obj/item/I, mob/user)
+	. = ..()
+	var/note = stripped_input(user, "What would you like the plaque to say? Default value is item's description.", "Frame Plaque")
+	if(note)
+		if(user.Adjacent(src))

```

Comment:
use canUseTopic, otherwise there's no stat check and they can set it after dying / ghosts can set it freely.

Reply:
maybe also add a meme user.is_literate check here too